e2e/config/config.go

```diff
@@ -180,6 +226,14 @@ func GetClusterSetName() string {
 	return config.ClusterSet
 }
 
+func GetNamespaces() Namespaces {
+	if config.Distro == distroK8s {
+		return k8sNamespaces
+	}
+
+	return ocpNamespaces
```

Comment:
This will saliently return ocpNamespaces if the validation is broken in future change, or if we add a third value. We want to return the right value from the distro value, or panic.

```go
switch config.Distro {
case distroK8s: return k8sNamesapces
case distroOcp: return ocpNamespaces
default:
    panic("invalid distro")
}
```

Reply:
thanks